rclpy/src/rclpy/_rclpy.c

```diff
@@ -834,7 +830,7 @@ rclpy_destroy_node_entity(PyObject * Py_UNUSED(self), PyObject * args)
   }
 
   rcl_node_t * node = (rcl_node_t *)PyCapsule_GetPointer(pynode, NULL);
-  rcl_ret_t ret;
+  rcl_ret_t ret = RCL_RET_ERROR;
```

Comment:
I have removed the commit from this PR and will handle my local linter somehow. So nothing needs to change here.